pkg/health/server/prober.go

```diff
@@ -379,7 +380,8 @@ func (p *prober) runProbe() {
 	for _, ips := range nodeIps {
 		ipCount += len(ips)
 	}
-	p.probeRateLimiter = rate.NewLimiter(Per(ipCount, p.server.Config.ProbeInterval), 1)
+	p.httpProbeRateLimiter = rate.NewLimiter(Per(ipCount, p.server.Config.ProbeInterval), 1)
+	p.icmpProbeRateLimiter = rate.NewLimiter(Per(ipCount*p.server.Config.ICMPReqsCount, p.server.Config.ProbeInterval), 1)
```

Comment:
I don't actually understand why we need to increase probing rate here.
`icmpPing` is still called  `ipCount` times. and `icmpPing` just performs `ICMPReqsCount` pings within `err = pinger.Run()`, no?

Reply:
Yep, you're right, the rate limiter should not be affected by the retries number in each single probe. I'll rework the PR, thanks. :+1: